src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.form-search {
+  margin-top: 20px;
+  position: relative;
+  align-items: center;
+}
+
+.input__search-bar {
+  outline: 0;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  width: 100%;
+  align-items: center;
+}
+
+.input__search-bar--big {
+  height: 70px;
+  padding-left: 62px;
+}
```

Comment:
we don't need the `--big` modifier, only `-small`, so move its styles to the general class